### PR TITLE
Introduce .prettierignore

### PR DESCRIPTION
**Pull request by wangannie**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"
### :cloud: :cloud: Description :cloud: :cloud:
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Added a `.prettierignore` file to ignore:
- `.github` so that any markdown templates are not formatted by prettier
- Generated files/directories:
  - `package-lock.json` because this file is generated and should never be directly modified
  - `.expo`, `ios/`, and `android/`

### Screenshots :calling:
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @leexesther at the bottom."
n/a

## :zap: How to review :zap:
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Adjusting spacing in `.github/pull_request_template.md` should not get reformatted by prettier! 

## :seedling: Next steps :seedling:
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
Adjust spacing in PR template @davidqing6432 

### Online sources :information_source:
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'
https://prettier.io/docs/en/ignore.html

### Related PRs :raised_hands:
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
Follows up addition & configuration of prettier from #8 

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @davidqing6432

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.github/
+package-lock.json
```
Comment (david-qing) [inline .prettierignore:4]: can you also add ios/ and android/ in here? i think those are coming up with prettier errors but shouldn't be modified

Comment (wangannie) [reply .prettierignore:4]: Good call -- updated!

**Review by david-qing — APPROVED** [review]
